index.md

```diff
@@ -56,11 +61,11 @@ Then, build and open the Angular app in the browser:
 ng serve -o
 ```
 
-We are ready to dive into Kendo UI Angular!
+We are ready to dive into Kendo UI for Angular! Next, we will [add some data](#add-data) to our app, and then we will bind this data to the [Kendo DropDownList](#add-a-dropdownlist) and the [Kendo Grid](#add-a-grid) components.
```

Comment:
I think you should announce the next steps because the average Angular developer probably hasn't learned much new things until this moment. Announcing the next steps should calm down the more impatient readers.

Reply:
Fair enough, although moving to the app folder is too basic step and I am afraid to announce it specifically. We can just move it next to `ng serve -o`?